src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  height: 408px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
```

Comment:
Padding should be defined using a multiple of 4 to maintain consistency with the spacing system.